### DISCUSSION TITLE
I have a problem with writing the interface definition of js-tokens. Please help.

### DISCUSSION THREAD

**Discussion by Longqin88888** [DefinitelyTyped/DefinitelyTyped#69149] [Request a new @types package]
I'm trying to write an interface definition for the 5.0.0 version of the js-tokens library. I've run into a problem that I hope can be solved.
js-tokens uses export.default to export a regular expression. The import method in JS is as follows:
var jsTokens = require("js-tokens").default.
More information can be seen https://github.com/lydell/js-tokens/tree/v5.0.0

**Comment by lydell** [comment]
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/69187#issuecomment-2099091959
